### PR TITLE
[Docs] big cleanup

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by OskarStark** [comment]
Thanks for reviewing @tifabien 👍

File: docs/reference/conditional_validation.rst [hunk 1]
```diff
@@ -28,17 +24,15 @@ object. The object can be used to check assertions against the model :
             // for maximum length constraint
             ->assertLength(array('max' => 100))
```
Comment (kunicmarko20) [inline docs/reference/conditional_validation.rst]: short array

File: docs/reference/conditional_validation.rst [hunk 2]
```diff
@@ -116,13 +109,11 @@ The method must accept two arguments:
  - ``ErrorElement``: the instance where assertion can be checked
  - ``value``: the object instance
 
-Example from the ``SonataPageBundle``
```
Comment (kunicmarko20) [inline docs/reference/conditional_validation.rst]: I think that `the` should be here? cc @greg0ire

File: docs/reference/doctrine_base_manager.rst [hunk 3]
```diff
@@ -5,15 +5,17 @@
...
+* ``Sonata\CoreBundle\Model\BaseEntityManager``
+* ``Sonata\CoreBundle\Model\BaseDocumentManager``
+* ``Sonata\CoreBundle\Model\BasePHPCRManager``
 
 Use it in your managers
 -----------------------
-You just have to extend ``Sonata\CoreBundle\Model\BaseEntityManager``, ``Sonata\CoreBundle\Model\BaseDocumentManager`` or ``Sonata\CoreBundle\Model\BasePHPCRManager`` in your managers, for instance:
 
-.. code-block:: php
+You just have to extend one of them in your managers::
```
Comment (kunicmarko20) [inline docs/reference/doctrine_base_manager.rst]: `extend one of the managers`?

File: docs/reference/conditional_validation.rst (hunk 2 above)
Comment (greg0ire) [reply docs/reference/conditional_validation.rst]: I think it should be either "Example from the Sonata page bundle" or "Example from `SonataPageBundle`"

File: docs/reference/doctrine_base_manager.rst (hunk 3 above)
Comment (OskarStark) [reply docs/reference/doctrine_base_manager.rst]: Done

**Pull request merged**